schema/geometry_schema.json

```diff
@@ -996,10 +1037,59 @@
 				"window_spacing",
 				"height",
 				"width",
+				"window_type",
```

Comment:
I notice we now have "window_definition_type", as well as "window_type". In the UI, these show up as "Type" and "Window Type":
![](https://trello-attachments.s3.amazonaws.com/58d428743111af1d0a20cf28/5a5f8009e07d38d0758796a9/a2144c431b587a269e81afadfe6c4e74/capture.png)

Could that be confusing for people? Perhaps we could call the Fixed/Operable distinction something else?

Reply:
change "Type" to "Mode". Add tests to catch this for backwards compatibility.